fastlane_bot/config/constants.py

```diff
@@ -11,3 +11,9 @@
     "fantom": 0.0003,
     "mantle": 0,
 }
+
+ETHEREUM = "ethereum"
+PANCAKESWAP_V3_NAME = "pancakeswap_v3"
+BUTTER_V3_NAME = "butter_v3"
+AGNI_V3_NAME = "agni_v3"
+CLEOPATRA_V3_NAME = "cleopatra_v3"
```

Comment:
It's already in network.py, why here too?

Reply:
@mikewcasale above asked to stop adding to network.py